ui/src/types/globalConfig/pages.ts

```diff
@@ -149,14 +149,27 @@ export const InputsPageTableSchema = z
     })
     .strict();
```

Comment:
```suggestion
    .transform((data) => ({ type: 'table', ...data } as const));
```

Reply:
or even better
```ts
type: z.literal('table').default('table')
```

Reply:
the same apply for service schema.
it will eliminate the need for guard function because you need to check the type by string comparation

Reply:
removed the type literal from schema